docs/resources/source_verification.md

```diff
@@ -54,7 +54,7 @@ Optional:
 - `github` (Attributes) (see [below for nested schema](#nestedatt--verification--github))
 - `gitlab` (Attributes) (see [below for nested schema](#nestedatt--verification--gitlab))
 - `hmac` (Attributes) (see [below for nested schema](#nestedatt--verification--hmac))
-- `json` (String, Sensitive)
```

Comment:
breaking change: I haven't added support for the catch-all JSON solution. However, as mentioned in a separate conversation, I feel the JSON solution doesn't add much value so not sure if we should try and continue support it?

Reply:
I think we should keep it, if anything to limit the breaking change and cover edge cases with the code gen?

Reply:
> if anything to limit the breaking change and cover edge cases with the code gen

I'm not 100% sure I follow you here. The main thing is this json solution doesn't cover any edge case, so that's the main thing. I can figure out a way to add support for it in the codegen if you'd like tho, so it's not a big deal.

Reply:
I'll let @leggetter comment here, I might be missing context

Reply:
@alexluong - I'm pretty confident we have an important customer using this feature, so we should keep it for now.

Reply:
@alexluong - ping on keeping `json` support because we have customers using it.